azure-functions-sdk/src/templates/new-app/local.settings.json.hbs

```diff
@@ -0,0 +1,10 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
```

Comment:
My only problem with these as default values is that it requires the storage emulator to run the app (which is Windows only I believe; there's Azurite too for other platforms, but I haven't used it).

I think we could have a section in the README on how to use the storage emulator, but I'm not comfortable with requiring it out of the box.

Reply:
I wasn't aware of that, I'll nuke it.